platforms/Windows/installer.wxs

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
+  <Bundle Name="Swift" Version="$(var.ProductVersion)" Manufacturer="swift.org" UpgradeCode="8c75f32a-7bdf-4c61-abf6-c7e1c4b8fbf6">
+    <BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.HyperlinkLicense">
+      <bal:WixStandardBootstrapperApplication LicenseUrl="" LogoFile="Resources/swift.png" SuppressOptionsUI="yes" SuppressRepair="no" />
```

Comment:
```
Resources/swift.png
```
Where is this image comping from?

Reply:
I yanked it from swift.org

Reply:
I should mention that if you have a better way to get the project logo, I'm happy to use that, I don't really have any issue with adjusting the content.

Reply:
Where is the Resources directory located, is it in swift-installer-scripts repo or another location?

Reply:
It is in the same repository, it is being created as part of this commit.

Reply:
https://developer.apple.com/swift/resources/ at the bottom of the page it contains logos.

Reply:
Are you able to use svg format?

Reply:
Thanks, thats where I originally grabbed the image from.  I don't think that the SVG format works for this, which is why I have the png.